packages/admin/cms-admin/src/generator/future/generateForm/generateFormField.ts

```diff
@@ -181,12 +181,36 @@ export function generateFormField({
         if (config.values) {
             throw new Error("custom values for staticSelect is not yet supported"); // TODO add support
         }
+
         const enumType = gqlIntrospection.__schema.types.find(
             (t) => t.kind === "ENUM" && t.name === (introspectionFieldType as IntrospectionNamedTypeRef).name,
         ) as IntrospectionEnumType | undefined;
         if (!enumType) throw new Error(`Enum type ${(introspectionFieldType as IntrospectionNamedTypeRef).name} not found for field ${name}`);
         const values = enumType.enumValues.map((i) => i.name);
-        code = `<Field
+
+        if (config.inputType === "radio") {
+            code = `
+            <RadioGroupField
+             ${required ? "required" : ""}
+             layout="column"
```

Comment:
I don't think we want the column layout by default in generated forms, at least according to the designs I have:

<img width="549" alt="image" src="https://github.com/user-attachments/assets/68b4ec9e-50ba-42a0-9af9-b850b2d8c11e">

```suggestion
```